apps/openssl.c

```diff
@@ -437,7 +429,9 @@ int main(int argc, char *argv[])
     BIO_free(bio_in);
     BIO_free_all(bio_out);
     apps_shutdown();
+#ifdef CRYPTO_MEBUG
```

Comment:
I think this is supposed to be `CRYPTO_MEBUG`. There are a whole bunch of similar typos all over.